Patches/Core/ThingDefs_Buildings/Buildings_Security.xml

```diff
@@ -429,6 +429,7 @@
 		<value>
 			<li Class="CompProperties_Explosive">
 				<explosiveDamageType>Bomb</explosiveDamageType>
+				<damageAmountBase>320</damageAmountBase>
```

Comment:
The IED is intended to roughly the equivalent of an 81mm HE shell wired up. The mortar does 156 damage, using 2 FSX per shell. So, this could be justified to do a tad more, but 320 is too much (though I understand you were adding it to address a different issue).